atlassian/rest_client.py

```diff
@@ -52,7 +53,45 @@ def __init__(
         cloud=False,
         proxies=None,
         token=None,
+        backoff_and_retry=False,
+        retry_error_matches=[(429, "Too Many Requests")],
```

Comment:
Tried this with Bitbucket Cloud today. Required the following change to make it work:
```suggestion
        retry_error_matches[(429, "Too Many Requests"),
                                         (429, "Unknown Status Code")],
```